etc/containers/systemd/tryton.container

```diff
@@ -0,0 +1,20 @@
+[Unit]
```

Comment:
This unit should have a: ''After=tryton-postgres.container` dependency so the database is started before the application.

Reply:
Yeah, I'll check how to do that with podman-systemd.unit.